src/components/pages/course-page/components/lesson/Lesson.module.scss

```diff
@@ -0,0 +1,62 @@
+@import 'src/styles/partials/colors';
+@import 'src/styles/partials/utils';
+
+.wrapper{
+  //padding: 6px 12px;
+  width: 100%;
+  padding: 0;
+  //margin: 20px 0;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.header{
+  background: $background-secondary;
+  width: 100%;
+  &:hover{
+    background: #D0E3DC;
+  }
+  &:active{
+    background: #B4D9CC;
+  }
+}
+
+.icon{
+  width: 32px;
+  height: 32px;
+  transition: transform .2s linear;
+  &Rotated {
+    width: 32px;
```

Comment:
In this case write `width:32px; height:32px` in `&Rotated` isn't relevant

Reply:
fixed